packages/components/scripts/wrapper-generator/AbstractWrapperGenerator.ts

```diff
@@ -76,8 +78,15 @@ export abstract class AbstractWrapperGenerator {
     const componentExports = this.relevantComponentTagNames
       .map((component) => {
         const componentSubDir = this.getComponentSubDir(component);
-        const componentFileName = this.getComponentFileName(component, true);
-        return `export * from './${componentSubDir ? componentSubDir + '/' : ''}${componentFileName}';`;
+        const componentFileNameWithoutExtension = path.parse(this.getComponentFileName(component)).name;
+        const getBarrelFileContent = this.getBarrelFileContent(
```

Comment:
Confusing naming especially since it is in fact an override for the default barrel file content.  
Therefore, it would make more sense to create a function called `getBarrelFileContent()` that returns exactly what happened before and then override this function in the `VueWrapprGenerator`.